README.md

```diff
@@ -10,6 +10,16 @@ fh-amqp-js
 
 FeedHenry AMQP client wrapper for communication with a Rabbit cluster.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | npm  |
+| Documentation:  | http://docs.feedhenry.com/v3/api/cloud_api.html  |
+| Issue tracker:  | https://issues.jboss.org/projects/FH/summary  |
+| Mailing list:   | [feedhenry-dev](https://www.redhat.com/archives/feedhenry-dev/) ([subscribe](https://www.redhat.com/mailman/listinfo/feedhenry-dev))  |
+| IRC:            | #feedhenry channel in the [freenode](http://freenode.net/) network.  |
```

Comment:
Do you think it would be useful to link the `#feedhenry` part to https://webchat.freenode.net/?channels=feedhenry?